ksqldb-execution/src/test/java/io/confluent/ksql/execution/plan/ForeignKeyTableTableJoinTest.java

```diff
@@ -43,29 +44,31 @@
   private ExecutionStep<KTableHolder<Struct>> left2;
   @Mock
   private ExecutionStep<KTableHolder<Struct>> right2;
+  @Mock
+  private Formats formats;
```

Comment:
Not 100% if those changes are correct. While the unit test passed, I don't really understand what we actually test here and how `EqualsTester` and `addEqualityGroup()` actually work

Reply:
This change is not correct. The whole point of the test in this file is to ensure that no one adds a new field that should be considered in the `equals()` and `hashCode()` methods without properly updating the methods. We need to update those methods in ForeignKeyTableTableJoin to consider the new field, and we need to update this test to catch any future regressions here.